open-policy-agent/authorization/disallow-host-namespaces/constraint.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sDisallowHostNamespaces
+metadata:
+  name: disallow-host-namespaces
+spec:
+  enforcementAction: dryrun
+  match:
+    kinds:
+      - apiGroups: [""]
+        kinds: ["Pod"]
+    excludedNamespaces: 
```

Comment:
it will be useful to find a way to maintain the  excludedNamespaces in one central location